openlibrary/templates/merge_queue/newest_comment.html

```diff
@@ -0,0 +1,39 @@
+$def with(comment=None, comment_str=None)
+
+$# Creates a single comment element for the librarian merge requests table.
+$#
+$# This template excepts comments in either object (`comment`) or string (`comment_str`) form.
+$# If a string is passed, it is assumed that the comment was submitted just now, and that the
+$# commenter is the currently logged-in patron.
+$#
+$# Parameters:
+$# comment : dict : Object that represents a single comment on a merge request.
+$#   comment['message'] : str : The comment
+$#   comment['username'] : str : Username of patron who left the comment
+$#   comment['timestamp'] : datetime : Timestamp of this comment
+$# comment_str : str : The comment
+
+$if comment_str:
+  $ comment_time = _('Just now')
+  $ commenter = ctx.user and ctx.user.key.split('/')[-1]
+  $ owner = 'owner'
+  $ message = comment_str
+$else:
+  $ comment_timestamp = to_datetime(comment['timestamp'])
+  $ date = comment_timestamp.strftime('%d %b %y')
+  $ time = comment_timestamp.strftime('%X')
+  $ comment_time = '%s %s' % (date, time)
+
+  $ commenter = comment['username']
+
+  $ username = ctx.user and ctx.user.key.split('/')[-1]
+  $ is_commenter = username == commenter
+  $ owner = 'owner' if is_commenter else ''
+
+  $ message = comment['message']
+
+<div class="mr-latest-comment">
+  <div class="mr-latest-comment__body">
+    $message
```

Comment:
![888485D7-F5ED-4027-87E8-6334F6E50F12](https://user-images.githubusercontent.com/3709715/227701429-2eb3b021-78d2-4cc9-b021-725064128db4.jpeg)

⛔ is because the GitHub editor expects all text files. To end with one and only one `return` key.

Reply:
roger that

Reply:
template is deleted